packages/api-client/package.json

```diff
@@ -25,7 +25,8 @@
         "prettier": "prettier --check '{src,public}/**/*.{js,jsx,ts,tsx,css,html,svg}'",
         "npm-publish": "npm publish --access public",
         "npm-version-patch": "npm version patch",
-        "npm-version-minor": "npm version minor"
+        "npm-version-minor": "npm version minor",
+        "nswag-generate": "nswag run src/nswag/nswag.json"
```

Comment:
This is the nswag command to generate the API proxies from the swagger file cc @keithrfung